docker-compose.yml

```diff
@@ -47,7 +47,7 @@ services:
       TNB_DATABASES: '{"default":{"CLIENT":{"host":"node-mongo","password":"${MONGO_INITDB_ROOT_PASSWORD}"}}}'
       TNB_CELERY_BROKER_URL: 'amqp://guest:guest@celery-broker:5672//'
     env_file: .env
-    command: poetry run celery -A node.config.celery worker --loglevel=INFO
+    command: poetry run celery -A node.config.celery worker --loglevel=INFO --pool=eventlet --concurrency=30
```

Comment:
let's have concurrency depend on max PV schedule length (read from env var that corresponds to django setting)

Reply:
Concurrency value is now same as TNB_NODE_SCHEDULE_CAPACITY environment value.